### PR TITLE
How to toggle password field in vuejs

### PR DESCRIPTION
**Pull request by tomboe311**
I have read and understood the Contributor's Certificate of Origin available at the end of 
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Thomas Boehringer <dev@tboehringer.de>

### CODE REVIEW

File: tutorials/how-to-toggle-password-field-in-vuejs/01.de.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+available_languages: ["en", "de"]
+header_img: "header-3"
+cta: "development"
+---
+
+## Einführung
+
+[Vue.js](https://vuejs.org/) ist ein clientseitiges Javascript-Webframework, vergleichbar mit React.js oder Angular.
+
+Dieses Tutorial zeigt wie in Vue.js 3.* wiederverwendbare "Components", anhand eines Passwortfelds mit einem Sichtbarkeit-Button, erstellt wird.
```
Comment (tobiasgruber1995) [inline tutorials/how-to-toggle-password-field-in-vuejs/01.de.md:23]:
```suggestion
Dieses Tutorial zeigt, wie in Vue.js 3.* wiederverwendbare "Components" anhand eines Passwortfelds mit einem Sichtbarkeit-Button erstellt werden.
```

File: tutorials/how-to-toggle-password-field-in-vuejs/01.de.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+cd vue-project/src
+```
+
+Hier erstellen wir mit dem Befehl `touch` eine neue Datei mit dem Namen `index.html`:
+
+```bash
+touch index.html
+```
+
+In dem Texteditor Deiner Wahl, öffne die neue HTML Datei und füge folgendes Basic HTML Code hinzu:
```
Comment (tobiasgruber1995) [inline tutorials/how-to-toggle-password-field-in-vuejs/01.de.md:49]:
```suggestion
In dem Texteditor deiner Wahl, öffne die neue HTML Datei und füge folgenden Basic HTML Code hinzu:
```

File: tutorials/how-to-toggle-password-field-in-vuejs/01.de.md [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+```bash
+mkdir components
+touch components/PasswordRevealer.js
+```
+
+### Step 3.2 - Component erstellen
+
+Diese Component `password-revealer` erstellt ein Passwortfeld mit einem Button zum Anzeigen des Passworts.
+* Erstellt die Variable `passwordType` und setzt den Wert auf `"password"`. 
+* Wenn der `<button` angeklickt wird, die Methode `switchVisibility` wird ausgeführt und der `type` des `<input>` wird zwischen `"text"` und `"password"` gewechselt.
```
Comment (tobiasgruber1995) [inline tutorials/how-to-toggle-password-field-in-vuejs/01.de.md:111]:
```suggestion
* Wenn der `<button>` angeklickt wird, wird die Methode `switchVisibility` ausgeführt und der `type` des `<input>` wird zwischen `"text"` und `"password"` gewechselt.
```

File: tutorials/how-to-toggle-password-field-in-vuejs/01.de.md [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+        `
+        <input id="password" :type="passwordType">
+        <button v-on:click="switchVisibility()">Show/Hide</button>
+        `
+})
+```
+
+### Step 3.3 - Component verwenden
+
+Am Ende des Bodys in die `index.html` wird die erstellte Datei aus Step 3.2 wie folgt eingebunden:
```
Comment (tobiasgruber1995) [inline tutorials/how-to-toggle-password-field-in-vuejs/01.de.md:136]:
```suggestion
Am Ende des Bodys wird die erstellte Datei aus Step 3.2 wie folgt in die `index.html` eingebunden:
```

File: tutorials/how-to-toggle-password-field-in-vuejs/01.en.md [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+
+
+**Prerequisites**
+
+* Basic knowledge about [Vue.js](https://vuejs.org/)
+* You will also need a basic knowledge of JavaScript, HTML, and CSS
+
+## Step 1 - Setting Up the Project
+
+At the beginning, the directory structure of the project is creating with the `mkdir` command:
```
Comment (tobiasgruber1995) [inline tutorials/how-to-toggle-password-field-in-vuejs/01.en.md:35]:
```suggestion
At the beginning, the directory structure of the project is created with the `mkdir` command:
```

File: tutorials/how-to-toggle-password-field-in-vuejs/01.en.md [hunk 6]
```diff
@@ -0,0 +1,229 @@
...
+```bash
+mkdir components
+touch components/PasswordRevealer.js
+```
+
+### Step 3.2 - Create Component
+
+This component `password-revealer` renders a password input field with a reveal button.
+* Create the variable `passwordType` and set the value to `"password"`.
+* When the `<button` is clicked, execute the method `switchVisibility` and change the `type` in `<input>` between `"text"` and `"password"`.
```
Comment (tobiasgruber1995) [inline tutorials/how-to-toggle-password-field-in-vuejs/01.en.md:110]:
```suggestion
* When the `<button>` is clicked, the method `switchVisibility` is executed and toggles the `type` in `<input>` between `"text"` and `"password"`.
```

File: tutorials/how-to-toggle-password-field-in-vuejs/01.en.md [hunk 7]
```diff
@@ -0,0 +1,229 @@
...
+
+```css
+.revealer {
+    color: #fff;
+    background-color: #6c757d;
+    border-color: #6c757d;
+}
+```
+
+The stylesheet is including in the `<head>` in the `index.html` file:
```
Comment (tobiasgruber1995) [inline tutorials/how-to-toggle-password-field-in-vuejs/01.en.md:174]:
```suggestion
The stylesheet is included in the `<head>` in the `index.html` file:
```

**Review by tobiasgruber1995 — CHANGES_REQUESTED** [review]

**Review by rook59 — APPROVED** [review]
German path is wrong otherwise it looks good to me

**Pull request merged**